test/test_parse_logcfg.c

```diff
@@ -915,6 +916,12 @@ void test_ssbmode(void **state) {
     assert_int_equal(trxmode, SSBMODE);
 }
 
+void test_operation_mode(void **state) {
+    int rc = call_parse_logcfg("OPERATING_MODE=CQ\n");
+    assert_int_equal(rc, 0);
+    assert_int_equal(cqmode, CQ);
```

Comment:
the initial value of `cqmode` is `CQ` (line 193), so it's better to test here for the other case.